src/components/input/input.component.scss

```diff
@@ -23,7 +23,7 @@ $input-icon-color: $color-grey-300;
 
   .ngx-input-wrap {
     position: relative;
-    display: inline-block;
+    display: block;
```

Comment:
This is the `.ngx-input-wrap` class.  This should not effect the ngx-input elements, which were and still are, block elements.